pkgs/top-level/python-packages.nix

```diff
@@ -7126,6 +7126,30 @@ in {
     };
   };
 
+  grammalecte = buildPythonPackage rec {
```

Comment:
This looks like an application and not a python library to me. If this is the case please move it out of `python-packages.nix` into its own file and reference it in `pkgs/top-level/all-packages.nix`